### ISSUE TITLE
Problems with adding jump_none=True to get_gradient_vector () and apply_gradient_vector()

### ISSUE THREAD

**Issue by kanadsen** [tum-pbs/ConFIG#4]
Hi, I have been using ConfigOperator() for some experimentation and came across the following issue:

- I have been applying this method for PDEs like NS equations and in some cases, the grad generated is not of the same shape. This causes an issue when the **torch.stack(grad)** is being called by the ConFIG_update() function. 
- I would recommend setting the gradients to 0 instead of skipping those gradients which is None since it worked for my case. Do have a look at this issue if possible.

**Comment by qiauil** [comment]
Hi, Kanadsen,

Sorry for the late reply; I need to find a way to subscribe to the issues on this repo...

Your advice is great; I will definitely add it to the new version. I will close this issue when it is integrated.

Thank you, and Cheers,

Qiang

**Comment by qiauil** [comment]
Hi, Kanadsen,

I have integrated the required feature into the new version. In your case, it should work by providing additional parameters to the `get_gradient_vector` and `apply_gradient_vector` as

```python
get_gradient_vector(...,none_grad_mode="zero")
apply_gradient_vector(...,none_grad_mode="zero")
```

Feel free to post if you have further issues.

Cheers,
Qiang
